### PR TITLE
feat(slider): introduce a slider compontent

### PR DESCRIPTION
**Pull request by steveoh**
the utah design system doesn't have any guidance but they do mention this component. I modeled the colors after the checkbox/radio button.

### CODE REVIEW

**Review by stdavis — COMMENTED** [review]
Cool! 

My only suggestion is that the thumb should have some sort of hover state to make it obvious that it's draggable especially since you are holding strong to the anti-pointer-cursor ~team~ cult. ;)

**Comment by steveoh** [comment]
> My only suggestion is that the thumb should have some sort of hover state to make it obvious that it's draggable especially since you are holding strong to the anti-pointer-cursor ~team~ cult. ;)

😆 Great point. hover states added.

File: packages/utah-design-system/src/components/Slider.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+export interface SliderProps<T> extends AriaSliderProps<T> {
+  label?: string;
+  thumbLabels?: string[];
+}
+
+function getTrackStyle(range: number, orientation?: string) {
+  if (orientation === 'vertical') {
+    return {
+      top: `${(1 - range) * 100}%`,
+      // transform: 'translate(-50%, -50%)',
```
Comment (stdavis) [inline packages/utah-design-system/src/components/Slider.tsx:48]: Want to dump this?

**Review by stdavis — APPROVED** [review]

File: packages/utah-design-system/src/components/Slider.tsx (hunk 1 above)
Comment (steveoh) [reply packages/utah-design-system/src/components/Slider.tsx]: yep, and removing the default width and height.

**Review by stdavis — APPROVED** [review]

**Review by stdavis — APPROVED** [review]

**Pull request merged**